src/Spectre.libClassifier.Tests/Spectre.libClassifier.Tests.vcxproj

```diff
@@ -88,7 +88,7 @@
       <SDLCheck>true</SDLCheck>
       <TreatWarningAsError>true</TreatWarningAsError>
       <PreprocessorDefinitions>GTEST_LANG_CXX11=1;_UNICODE;UNICODE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
-      <AdditionalIncludeDirectories>$(SolutionDir);%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>$(SolutionDir);$(SolutionDir)\..\dep\gsl\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
It'd be better to push this into our default Property Sheet, since this change is present in almost each and every project.

Reply:
Addressed in 177a5698cb36451cae935d11d78cfeec5503989b.